packages/labs/scoped-registry-mixin/src/test/scoped-registry-mixin_test.ts

```diff
@@ -24,6 +24,14 @@ class SimpleGreeting extends ScopedRegistryHost(LitElement) {
     return {name: {type: String}};
   }
 
+  static get styles() {
```

Comment:
Is there a reason why SimpleGreeting is a ScopedRegistryHost? It doesn't register any of its own children. I wonder if we could get better coverage of scenarios if SimpleGreeting just extended LitElement, and move these styles for the style test to ScopedComponent.

Reply:
makes sense. I did the change.

Reply:
done